src/Sylius/Bundle/CoreBundle/Controller/ProductController.php

```diff
@@ -283,4 +314,38 @@ private function renderResults(
 
         return $this->handleView($view);
     }
+
+    private function renderArchetypeResults(
+        ArchetypeInterface $archetype,
+        Pagerfanta $results,
+        $template,
+        $page,
+        $facets = null,
+        $facetTags = null,
+        $filters = null,
+        $searchTerm = null,
+        $searchParam = null,
+        $requestMethod = null
+    )
+    {
```

Comment:
Coding standards ;) If you put arguments in column, parenthesis and brace should be in the same line.